tests/test_backend.py

```diff
@@ -2078,5 +2078,17 @@ def test_space_to_batchnd(self):
         _ = tf.space_to_batch_nd(input_x, block_size, pad, name=_TFOUTPUT)
         self._run_test_case([_OUTPUT], {_INPUT: input_val})
 
+    @check_opset_min_version(10, "is_inf")
+    def test_isinf(self):
+        x_val1 = np.array([1.0, 2.0, -3.0, -4.0], dtype=np.float32).reshape((2, 2))
```

Comment:
is double type support?
can we try input with a few different ranks, not just (2, 2)

Reply:
Yes, double type is also supported.

Reply:
i mean different rank, e.g. 1d, 2d, 3d tensor.